ui/admin/app/routes/scopes/scope/host-catalogs/new.js

```diff
@@ -17,21 +17,24 @@ export default class ScopesScopeHostCatalogsNewRoute extends Route {
 
   // =methods
 
-  /**
-   * Rollback/destroy any new, unsaved instances from this route before
-   * creating another.
-   */
-  beforeModel() {
-    if (this.currentModel?.isNew) this.currentModel.rollbackAttributes();
-  }
-
   /**
    * Creates a new unsaved host catalog belonging to the current scope.
+   * Also rollback/destroy any new, unsaved instances from this route before
+   * creating another, but reuse name/description if available.
    * @return {HostCatalogModel}
    */
   model() {
     const scopeModel = this.modelFor('scopes.scope');
-    return this.store.createRecord('host-catalog', { scopeModel });
+    let name, description;
+    if (this.currentModel?.isNew) {
```

Comment:
In the case the model is not new, `name` and `description` will keep being undefiend, right?

Reply:
Correct, in which case there is no effect.